### ISSUE TITLE
mysql-python not available in OSX

### ISSUE THREAD

**Issue by Jorge-C** [ContinuumIO/anaconda-issues#254]
`mysql-python` isn't available in OSX:
```
bash-4.1$ conda search --platform osx-64 mysql-python
Fetching package metadata: ...
bash-4.1$ conda search --platform linux-64 mysql-python
Fetching package metadata: ...
mysql-python              *  1.2.3                    py27_0  defaults        
```
From a bit of googling it seems it should also work in OSX.

**Comment by tswicegood** [comment]
@ilanschnell any thoughts here?

**Comment by ilanschnell** [comment]
Yes, mysql-python works on OSX, there just isn't a conda package in repo.continuum.io, but maybe someone created a package on binstar.org

**Comment by Jorge-C** [comment]
Thanks! I don't really need that package any more, but not having it available in one platform where it works seems like a bug :)

When I created this issue, it caused some extra problems to port a conda package from linux to OSX.

**Comment by tswicegood** [comment]
@ilanschnell any idea how that ended up in `defaults` with only one platform?  I thought the packages that were there were generally there for all platforms.

**Comment by ilanschnell** [comment]
Generally yes, for all platforms.  But in this case, we needed it for a particular use case, and it was just created for that one platform.  This was even before binstar.org.  In some other cases, there are packages which are only available for Python 2.7, even though they could be created for Python 3.4 as well.  Again, we try to make the packages available for all platforms and all Python versions, but if they are hard to build and the use case (or customer) only needs something in particular, we stop there.
